packages/x-adapter-platform/src/mappers/requests/experience-controls-request.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { schemaMapperFactory } from '@empathyco/x-adapter';
+import { ExperienceControlsRequest } from '@empathyco/x-types';
+/* eslint-disable max-len */
```

Comment:
Wrap all the imports when you're using the eslint disable block, even if some don't need it.